mkdocs.yml

```diff
@@ -12,6 +12,7 @@ nav:
   - writing-rules/testing-rules.md
   - running-rules.md
   - ignoring-findings.md
+  - running-in-ci.md
```

Comment:
Just noting here that we should look for other places where references to CI docs may need to be updated.